src/constants/translations/en/offer-card-square.json

```diff
@@ -0,0 +1,6 @@
+{
+    "chips": {
```

Comment:
we don't create a new file for each component, and I think such translations already exist

Reply:
> we don't create a new file for each component, and I think such translations already exist

understand, removed it